### PR TITLE
feat: add count aggregation

### PR DESCRIPTION
**Pull request by kevinjnguyen**
Adds the count aggregation to the new python implementation. Adds `count` in #662 .

### CODE REVIEW

File: python/pytests/aggregation_test.py [hunk 1]
```diff
@@ -48,3 +48,26 @@ def test_sum_since_true(source, golden) -> None:
         }
     )
     golden.jsonl(m_sum_since_true)
+
+
+def test_count_unwindowed(source, golden) -> None:
```
[the remark this answers is not in the document]
Comment (jordanrfrazier) [reply python/pytests/aggregation_test.py:53]: I'd rather have some sort of separation. That way when we add new functions or capabilities, it's much easier to reference a single file or block and say "Ok, here's the list of tests I at least should have for my new aggregation". 

Also, once we move over the sparrow `e2e` tests here, we'll have a _massive_ number of lines.

[the remark this answers is not in the document]
Comment (jordanrfrazier) [reply python/pytests/aggregation_test.py:53]: I think it's fine for now -- once you finish the aggregations you can have a PR to move them into separate test files if we want to go that route. Or you can do it now, if you like.

**Review by jordanrfrazier — APPROVED** [review]

File: python/pysrc/kaskada/_timestream.py [hunk 2]
```diff
@@ -993,6 +993,26 @@ def last(self, window: Optional[kd.windows.Window] = None) -> Timestream:
         """
         return _aggregation("last", self, window)
 
+    def count(self, window: Optional[kd.windows.Window] = None) -> Timestream:
```
Comment (bjchambers) [inline python/pysrc/kaskada/_timestream.py:996]: Add to `aggregation.md` to show up in docs.

File: python/pytests/aggregation_test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinjnguyen) [reply python/pytests/aggregation_test.py]: I went with this route and split them out.

**Pull request merged**